modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -30,6 +30,13 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+[discrete]
+== Procedure Verification Steps
+This section is recommend but optional. Provide the user one or more steps to verify that the procedure provided the intended outcome. This may consist of:
```

Comment:
recommend -> recommended

Provide the user _with_...

Reply:
I agree that this section should be optional, but I don't think we should recommend it. In many cases, the verification should just be the final step (or steps) in the procedure.

Reply:
Thanks for spotting the typos :) I was in too much of a hurry.
Removed "... recommended but ..."